src/mocks/data/portefolje.ts

```diff
@@ -394,70 +392,68 @@ export function hentArbeidslisteForBruker(fnr: {fodselsnummer: any}) {
     return lagArbeidsliste('1', fnr);
 }
 
-export function hentMockPlan() {
+export function hentMockPlan(): MoteData[] {
+    const deltaker1 = {fornavn: 'john', etternavn: 'johnson', fnr: '123'};
+    const deltaker2 = {fornavn: 'Mars', etternavn: 'Testson', fnr: '456'};
+    const deltaker3 = {fornavn: 'Ada', etternavn: 'Kadabra', fnr: '11223312345'};
+
+    const now = new Date();
     const omToDager = new Date();
     omToDager.setDate(omToDager.getDate() + 4);
+
     function randomDate(start, end) {
-        return new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime()));
+        return new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime())).toString();
+    }
+    function motedataRandomDager(antallMoter: Number): MoteData[] {
```

Comment:
SonarCloud foreslår å bytte til primitves `number` i stedet:
```suggestion
    function motedataRandomDager(antallMoter: number): MoteData[] {
```